conf/messages.en

```diff
@@ -212,23 +212,23 @@ tariff.declaration.destinationCountry.1.common.linkText.0 = data element 5/8 - C
 tariff.declaration.destinationCountry.2.common.text = If you select ''Jersey'' or ''Guernsey'', your declaration will be updated to {0}.
```

Comment:
Wondering if it would be worth to change these single quotes to open/close apostrophes.

Reply:
Open apostrophe **‘**
Close apostrophe **’**